app/src/main/java/com/navapp/navigation/util/SimpleUponArrival.java

```diff
@@ -1,31 +0,0 @@
-package com.navapp.navigation.util;
```

Comment:
YOU WORKED ON DEV

Reply:
HOW DID THIS HAPPEN TO MEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE I MADE A MISTAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAKE

Reply:
THATS WHY WE NEED PROTECTIONS

Reply:
are you sure about that?